src/LodeRunner.UI/src/components/IncompleteTestRuns/styles.css

```
@@ -0,0 +1,27 @@
+.incompletetestruns {
+  border-top: #156064 2px solid;
+  margin-top: 1em;
+  padding-top: 1em;
+}
+
+.incompletetestruns > a.navigation {
+  display: block;
+  text-align: center;
+}
+
+.incompletetestruns-item {
+  text-align: left;
+  margin: 6px 0;
+  width: 100%;
+  border: none;
+  border-radius: 4px;
+  padding: 1em;
+  background-color: var(--c-primary-dark);
+  color: var(--c-neutral-lightest);
+  font-weight: bold;
+}
```

this needs to be flex for margins to overlap. without it, they appear as 12px which doesn't match clients and configs
```suggestion
.incompletetestruns-item {
  text-align: left;
  display: flex; 
  margin: 6px 0;
  width: 100%;
  border: none;
  border-radius: 4px;
  padding: 1em;
  background-color: var(--c-primary-dark);
  color: var(--c-neutral-lightest);
  font-weight: bold;
}
```